src-clock/mtime_clock_stubs.c

```diff
@@ -51,14 +54,14 @@ CAMLprim value ocaml_mtime_clock_elapsed_ns (value unit)
   static uint64_t start = 0L;
   if (start == 0L) { start = mach_absolute_time (); }
```

Comment:
Just for the record there is a bug here. You forgot to update that to `mach_continuous_time`.